### DISCUSSION TITLE
Engagement

### DISCUSSION THREAD

**Discussion by IRobot1** [angular-threejs/angular-three#18] [General]
Just some feedback on your repositories

- this [home page](https://github.com/angular-threejs/angular-three) is not engaging and assumes someone will look in the sub-folders.  Perhaps a link to your blog
- you abandoned your last angular-three project.  What's the long term plan for this project?  How many more re-writes?  Transparency would help
- the migration documentation from previous angular-three to this version is poor.  Have you tried taking anyone elses project and migrating it?  
- have you tried promoting on [threejs](https://threejs.org/docs/index.html#manual/en/introduction/Libraries-and-Plugins) or forums?

I was a big fan, until you abandoned the original version.  The effort to migrate my original example projects isn't worth it.

**Comment by IamSarki** [comment]
It will also help to know if there are any plans for Angular 17, I am starting a new Angular project and I would like to use the latest version
